sandbox/src/Bitcoin.tsx

```diff
@@ -232,6 +232,90 @@ function BtcSignPSBT({ bb02 }: Props) {
   );
 }
 
+function BtcSignMessage({ bb02 }: Props) {
+  const [coin, setCoin] = useState<bitbox.BtcCoin>('btc');
+  const [simpleType, setSimpleType] = useState<bitbox.BtcSimpleType>('p2wpkhP2sh');
+  const [keypath, setKeypath] = useState("m/49'/0'/0'/0/0");
+  const [message, setMessage] = useState('message');
+  const [running, setRunning] = useState(false);
+  const [result, setResult] = useState<bitbox.BtcSignMessageSignature | undefined>();
+  const [err, setErr] = useState<bitbox.Error>();
+
+  const simpleTypeOptions: bitbox.BtcSimpleType[] = ['p2wpkhP2sh', 'p2wpkh', 'p2tr'];
+
+  const script_config: bitbox.BtcScriptConfig = { simpleType };
+  const script_config_w_keypath: bitbox.BtcScriptConfigWithKeypath = { scriptConfig: script_config, keypath: keypath };
+
+  const stringToUint8Array = (str: string) => {
+    const arr = new Uint8Array(str.length);
+    for (let i = 0; i < str.length; i++) {
+        arr[i] = str.charCodeAt(i);
+    }
+    return arr;
+  }
+
+  const parsedResult = result ? JSON.stringify(result, undefined, 2) : '';
+
+  const submitForm = async (e: FormEvent) => {
+    e.preventDefault();
+    setRunning(true);
+    setResult(undefined);
+    setErr(undefined);
+    try {
+      const signature = await bb02.btcSignMessage(coin, script_config_w_keypath, stringToUint8Array(message));
+      setResult(signature);
+    } catch (err) {
+      setErr(bitbox.ensureError(err));
+    } finally {
+      setRunning(false);
+    }
+  }
+
+  return (
+    <div>
+      <h4>Sign Message</h4>
+      <form className="verticalForm" onSubmit={submitForm}>
+        <label>
+          Coin
+          <select value={coin} onChange={(e: ChangeEvent<HTMLSelectElement>) => setCoin(e.target.value as bitbox.BtcCoin)}>
+            {btcCoinOptions.map(option => <option key={option} value={option}>{option}</option>)}
+          </select>
+        </label>
+        <label>
+          Simple Type
+          <select value={simpleType} onChange={(e: ChangeEvent<HTMLSelectElement>) => setSimpleType(e.target.value as bitbox.BtcSimpleType)}>
+            {simpleTypeOptions.map(option => <option key={option} value={option} disabled={option === 'p2tr' && (coin === 'ltc' || coin === 'tltc')}>{option}</option>)}
+          </select>
+        </label>
+        <label>
+          Keypath
+        </label>
+        <textarea value={keypath} onChange={e => setKeypath(e.target.value)} rows={1} cols={80} />
```

Comment:
I had set the rows to 1 and thought it would serve the same purpose, but after reading this I saw it was not the same. Adjusted it to the `input` tag.